### PR TITLE
typos in en.json

### PR DESCRIPTION
**Pull request by Jibec**

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -124,9 +124,9 @@
     "global_settings_bad_choice_for_enum": "Bad value for setting {setting:s}, received {received_type:s}, except {expected_type:s}",
     "global_settings_bad_type_for_setting": "Bad type for setting {setting:s}, received {received_type:s}, except {expected_type:s}",
     "global_settings_cant_open_settings": "Failed to open settings file, reason: {reason:s}",
-    "global_settings_cant_serialize_setings": "Failed to serialzed settings data, reason: {reason:s}",
+    "global_settings_cant_serialize_setings": "Failed to serialized settings data, reason: {reason:s}",
```
Comment (opi) [inline locales/en.json]: "Failed to serialize" no ?

btw, there is a typo on the key too : se*t*ings.

File: locales/en.json [hunk 2]
```diff
@@ -124,9 +124,9 @@
...
     "global_settings_cant_write_settings": "Failed to write settings file, reason: {reason:s}",
-    "global_settings_key_doesnt_exists": "The key '{settings_key:s}' doesn't exists in the global settings, you can see all the available keys by doing 'yunohost settings list",
+    "global_settings_key_doesnt_exists": "The key '{settings_key:s}' doesn't exists in the global settings, you can see all the available keys by doing 'yunohost settings list'",
```
Comment (opi) [inline locales/en.json]: - "does not exist" without trailing s. (key and value).
- I would remove "you can", but it's more a personal taste.

File: locales/en.json (hunk 1 above)
Comment (M5oul) [inline locales/en.json]: > "Failed to serialize" no ?

Yep.

> btw, there is a typo on the key too : setings.

Nope, [there is two `t`](http://www.linguee.com/english-french/search?source=auto&query=settings).

Comment (opi) [inline locales/en.json]: > Nope, there is two t.

that's what I said...

Comment (M5oul) [inline locales/en.json]: ah, on the key. I thought on the string.

**Pull request merged**